flow/layers/container_layer.h

```diff
@@ -44,6 +44,39 @@ class ContainerLayer : public Layer {
   FML_DISALLOW_COPY_AND_ASSIGN(ContainerLayer);
 };
 
+class SystemCompositedContainerLayer : public ContainerLayer {
```

Comment:
Maybe call this `FuchsiaSystemCompositedContainerLayer` because in the future there might also be `AndroidSystemCompositedContainerLayer` or `IosSystemCompositedContainerLayer` which may not have elevations.